app/models/event_blacklist.rb

```diff
@@ -0,0 +1,76 @@
+class EventBlacklist < ActiveRecord::Base
+  default_value_for :enabled, true
+  after_save        :queue_sync_filtered_events
+  after_destroy     :queue_sync_filtered_events
+  after_destroy { |r| _log.info("Event filter [#{r.event_name}] for provider #{r.provider_model} with ID [#{r.ems_id}] has been deleted by user [#{self.class.current_userid}]") }
```

Comment:
Do we need this?  If so, perhaps it should go into the AuditLog instead of evm log?